vaadin-select-flow-parent/vaadin-select-flow/src/main/java/com/vaadin/flow/component/select/generated/GeneratedVaadinSelect.java

```diff
@@ -357,38 +357,6 @@ protected void setLabel(String label) {
         getElement().setProperty("label", label == null ? "" : label);
     }
 
-    /**
-     * <p>
-     * Description copied from corresponding location in WebComponent:
-     * </p>
-     * <p>
-     * The current required state of the select. True if required.
-     * <p>
-     * This property is not synchronized automatically from the client side, so
-     * the returned value may not be the same as in client side.
-     * </p>
-     *
-     * @return the {@code required} property from the webcomponent
-     */
-    protected boolean isRequiredBoolean() {
```

Comment:
Since removing protected API is considered breaking, I guess this can't be merged before the next major.